README.md

```diff
@@ -1,24 +1,262 @@
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* ...
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Only Admin can add or delete Cars from the website.
+- Users can create accounts and log in securely to access the booking functionalities.
+- Choose from a wide range of available cars based on your preferences and requirements.
+- Easily book a car for your desired date and time and manage your bookings efficiently.
+- User can Cancel the booking.
+- Website is responsive and is designed to work nicely for different devices and screens.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- React Frontend -->
+
+## <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/539px-React-icon.svg.png" width="23" height="20"/> React Frontend <a name="react-frontend"></a>
+
+- Frontend part of the project [Fast-Lane-Racers-frontend](https://github.com/muneeb3123/Fast-Lane-Racers-front-end)
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Code Editor (Vs Code)
+- Terminal
+- Install ruby on your computer
+- Install rails on your computer
+- Node.js
+- PostgreSQL
+
+### Setup
+
+Clone these repositories by using the command line to your desired folder:
+
+<u>**For Backend**</u>
+
+```sh
+git clone https://github.com/muneeb3123/Fast-Lane-Racers-back-end
+
+cd Fast-Lane-Racers-back-end
+```
+
+<u>**For Frontend**</u>
+
+```sh
+git clone https://github.com/muneeb3123/Fast-Lane-Racers-front-end
+
+cd Fast-Lane-Racers-front-end
+```
+
+### Install
+
+Install this project with:
+
+<u>**For Backend**</u>
+
+```sh
+  bundle install
+```
+
+<u>**For Frontend**</u>
+
+```sh
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+<u>**For Backend**</u>
+
+```sh
+  Set up JWT secret key key as follow:-
+  rails secret (copy secret_key)
+
+  Linux/Mac user:-
+  export DEVISE_JWT_SECRET_KEY=secret_key
+
+  Window user:-
+  set DEVISE_JWT_SECRET_KEY=secret_key
+  or
+  $env:DEVISE_JWT_SECRET_KEY="secret_key"
+
+
+  rails server
+```
```

Comment:
- [ ] When I try to set up the project and run it according to your instructions, this happens:

The env secret_key has been setup on ENV, according to instructions:

![image](https://github.com/muneeb3123/Fast-Lane-Racers-back-end/assets/57421823/70472500-a36e-4a35-aa28-5585bed2ab29)


The server running fails, and I can't create the database because of the same reason:

![image](https://github.com/muneeb3123/Fast-Lane-Racers-back-end/assets/57421823/e3b8e011-cd77-4a66-bd7a-6d42648876e3)

Kindly, improve your instructions and/or include the master.key file so your project is able to be set up and run correctly.

_note: This means I wasn't able to assert the project's functionality.  This will be taken a look at the next review._